docs/pages/ESSupport.md

```diff
@@ -854,3 +854,45 @@ This analyzer checks every JavaScript file for occurences of a `sap/ui/core/Core
 ### JSDoc
 
 The UI5 Tooling offers a JSDoc build, which is enhanced by UI5 specific JSDoc features.
+
+Following artefacts are part of the JSDoc build:
+
+- [jsdocGenerator.js](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/jsdocGenerator.js)
+  Executes the actual JSDoc build and creates the `api.json`.
+- [sdkTransformer.js](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/sdkTransformer.js)
+  Transforms the `api.json` for usage in a UI5 SDK.
+- [apiIndexGenerator.js](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/apiIndexGenerator.js)
+  Compiles API index resources from all `api.json` resources available in the given test resources directory.
+  The resulting index resources (e.g. `api-index.json`,  `api-index-deprecated.json`,
+  `api-index-experimental.json` and `api-index-since.json`) are mainly to be used in the UI5 SDK.
+
+#### JSDoc: UI5 SDK
+
+The UI5 SDK can be build locally. To get more insights about the local UI5 SDK build setup, have a look at the [Developer Guide](https://github.com/SAP/openui5/blob/master/docs/developing.md#building-the-openui5-sdk-demo-kit).
+
+Currently, the resources needed for the UI5 SDK build are stored in [openui5](https://github.com/SAP/openui5/tree/master/lib/jsdoc) and in [ui5-builder](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc). This is needed because these files are not part of the `sap.ui.core` library artefact, so building JSDoc for any library has no access to the needed resources. Therefore it is necessary to have these resources also available in the *ui5-builder*. This might chance in future.
+
+Following artefacts are part of the UI5 SDK build:
+
+- [createIndexFiles.cjs](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/lib/createIndexFiles.cjs): Creates several index files (e.g. `sap-ui-version.json`).
```

Comment:
This is the script used by the `apiIndexGenerator` processor. It **uses** the `sap-ui-version.json` but doesn't create it.

Reply:
Done